travis-autobuild-extensions.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ "$TRAVIS_REPO_SLUG" == "new-xkit/XKit" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]; then
+  echo -e "Rebuilding extension dist...\n" &&\
+
+  cd $HOME &&\
+  git config --global user.email "travis@travis-ci.org" &&\
+  git config --global user.name "travis-ci" &&\
+  git clone https://"$GH_TOKEN"@github.com/new-xkit/XKit &&\
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1437010/10323249/0e20d4ea-6c82-11e5-877c-91bdeaa94026.png)
this is where I added the GH_TOKEN